source/appModules/windowsinternal_composableshell_experiences_textinput_inputapp.py

```diff
@@ -221,7 +227,7 @@ def event_nameChange(self, obj, nextHandler):
 		or (self._recentlySelected is not None and self._recentlySelected in obj.name)):
 			return
 		# The word "blank" is kept announced, so suppress this on build 17666 and later.
-		if winVersion.winVersion.build > 17134:
+		if winVersion.getWinVer().build > 17134:
 			# In build 17672 and later, return immediatley when element selected event on clipboard item was fired just prior to this.

```

Comment:
immediatley -> immediately While it is not a mistake from this PR it makes sense to fix this while at it.